src/components/MovieCard/MovieCard.tsx

```diff
@@ -12,7 +12,9 @@ export const MovieCard: React.FC<Props> = ({ movie }) => (
       <figure className="image is-4by3">
         <img
           data-cy="moviePoster"
-          src={movie.imgUrl}
+          src={movie.imgUrl === 'N/A'
```

Comment:
you can put it here
![image](https://github.com/mate-academy/react_movies-list-fetch-movies/assets/91411692/24acd6e8-d44a-4961-9fdf-6010bee30dd9)